src/NodeJs/lib/jsonStream.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+const fs = require('fs');
+
+// We are using separated file descriptor to read JSON documents.
+// Number of the custom file descriptor is loaded from ENV (set by PHP), ... fallback is STDIN.
+let jsonStreamFd;
+if (process.env.JSON_STREAM_FD !== undefined) {
+  jsonStreamFd = parseInt(process.env.JSON_STREAM_FD, 10);
```

Comment:
Bude to fungovat podobne ako cosmosdb extractor, len obratene, PHP bude posielat spravy (a nie citat) do NodeJs procesu (kedze pre NodeJs mame k dispozicii dobru kniznicu). .... implementacia takto zaberie menej casu ako keby som to mal cele napisat v NodeJs (nemame sablonu komponenty), ... alebo v Python (nemam az take skusenosti).